sdk/cosmos/azure-cosmos/azure/cosmos/errors.py

```diff
@@ -63,3 +63,13 @@ class CosmosResourceExistsError(ResourceExistsError, CosmosHttpResponseError):
 
 class CosmosAccessConditionFailedError(CosmosHttpResponseError):
     """An error response with status code 412."""
+
+
+class ClientTimeoutError(AzureError):
+    """An operation failed to complete within the specified timeout."""
+
+    def __init__(self, **kwargs):
+        message = "Client operation failed to complete within specified timeout."
+        self.response = None
+        self.history = None
+        super(ClientTimeoutError, self).__init__(message, **kwargs)
```

Comment:
Every other class in this module has `Cosmos` in the name. Is this timeout error really specific to cosmos? If so, suggest a similar name for consistency